tools/batch/README.md

```diff
@@ -13,21 +13,55 @@ python3 submit-job.py \
 --wait
 ```
 
+# Updating the Docker for AWS Batch.
+
+Our current batch job dockers are in 747303060528.dkr.ecr.us-east-1.amazonaws.com/gluon-nlp-1. To
+update the docker:
+- Update the Dockerfile
+- Make sure docker and docker-compose, as well as the docker python package are installed.
+- Export the AWS account credentials as environment variables
+- CD to the same folder as the Dockerfile and execute the following:
+
+```
+# this executes a command that logs into ECR.
+$(aws ecr get-login --no-include-email --region us-east-1)
+
+# builds the docker image use the command in docker
+
```

Comment:
We may need to clarify that the `docker` here is `gluon-nlp/tools/docker/README.md`